frontend/src/lib/delete-policy.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright Contributors to the Open Cluster Management project
+
+import { Policy, PolicyApiVersion, PolicyKind, IResource, ResourceError, ResourceErrorCode, PolicyPluralKind } from '../resources'
+// import { PlacementBindingApiVersion, PlacementBindingKind } from '../resources/placement-binding'
+
+import { deleteResources } from './delete-resources'
+
+export function deletePolicy(policy: Policy) {
+    const resources: IResource[]  = [
+        {
+            apiVersion: PolicyApiVersion,
+            kind: PolicyKind,
+            plural: PolicyPluralKind,
+            metadata: { name: policy.metadata.name!, namespace: policy.metadata.namespace! },
+            
+        },
+    ]
+    // resources.push({
+    //     apiVersion: PlacementBindingApiVersion,
+    //     kind: PlacementBindingKind,
+    //     metadata: { name: policy.metadata.name },
+    // })
+    const deleteResourcesResult = deleteResources(resources)
+
+    return {
```

Comment:
Think there is a helper function for this...

Reply:
not able to locate helper function for the promise, can wait to add that in this PR or next iteration.